api.md

```diff
@@ -431,12 +434,72 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`](#chunk_relation_size)
 function.
 
-**Space partitions**: In most cases, it is advised for users not to use
+**Space partitions:** In most cases, it is advised for users not to use
 space partitions. The rare cases in which space partitions may be useful
 are described in the [add dimension][] section.
 
 ---
 
+## create_distributed_hypertable() [](create_distributed_hypertable)
```

Comment:
Although this shares arguments with `create_hypertable` it should have its own arguments section, etc., even if is a duplicate.  This prevents the user from having to go back and forth on the page to utilize the api.

Reply:
I was wondering if I could get away with this.  Copied the sections down and cleaned up the wording of replication_factor and data_nodes (which become much cleaner in this context).